### DISCUSSION TITLE
OBS and osu!lazer framerate issues

### DISCUSSION THREAD

**Discussion by squareSphere29** [ppy/osu#20779] [Q&A] [answered]
Before I open a bug issue,
Running OBS with osu!lazer causes some performance drop.
I've tried osu with fullscreen/borderless, same thing. OBS with/without preview doesnt change anything.
The difference is noticeable during gameplay.

PC:
AMD Ryzen 7 5800H
NVIDIA RTX 3060
16 GB RAM DDR4 3200
Windows 10

![with_OBS_running](https://user-images.githubusercontent.com/18124696/196024462-74e73451-3fd0-46ba-a36d-89afec02c94d.png)
With OBS

![without_OBS](https://user-images.githubusercontent.com/18124696/196024465-47d0a539-7845-4f81-a15f-2fb8f598f78f.png)
Without OBS

**Comment by peppy** [comment, marked as answer]
The only difference if your draw rate being capped to your refresh rate. This is likely being done by OBS or something else. It's a non-issue.
